src/video_core/texture_cache/texture_cache.h

```diff
@@ -1492,4 +1619,30 @@ bool TextureCache<P>::IsFullClear(ImageViewId id) {
            scissor.max_y >= size.height;
 }
 
+template <class P>
+void TextureCache<P>::UpdateFramebufferReferences() {
+    for (u32 i = 0; i < slot_images.Size(); ++i) {
+        const SlotId id{i};
+        if (slot_images.IsIndexFree(id)) {
+            Image& image = slot_images[id];
+            const bool in_use =
+                std::ranges::any_of(image.image_view_ids, [this](ImageViewId& view) {
+                    return active_framebuffer_imageviews.contains(view);
+                });
+
+            if (in_use) {
+                // Update the times for the aliases and views for this image,
+                // as they may not be directly referenced otherwise (i.e pause menus)
+                image.last_access_time = current_time;
+                std::ranges::for_each(image.image_view_ids, [this](ImageViewId& view) {
+                    slot_images[slot_image_views[view].image_id].last_access_time = current_time;
+                });
+                for (auto& alias : image.aliased_images) {
```

Comment:
```suggestion
                for (const auto& alias : image.aliased_images) {
```

Since the alias is only being queried